src/frontend/web_application/src/components/ImportContactForm/presenter.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { v1 as uuidV1 } from 'uuid';
+import { FieldErrors } from '../form';
+import Button from '../Button';
+import Icon from '../Icon';
+
+import './style.scss';
+
+class ImportContactForm extends Component {
+  static propTypes = {
+    uploadFile: PropTypes.func,
+    onCancel: PropTypes.func,
+    __: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    uploadFile: null,
+    onCancel: null,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      files: [],
+      errors: [],
+    };
+
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onResetForm = this.onResetForm.bind(this);
+    this.renderButtons = this.renderButtons.bind(this);
+  }
+
+  onInputChange(ev) {
+    const file = ev.target.files[0];
+    this.validate(file);
+  }
+
+  onResetForm() {
+    document.getElementById('import-contact-form').reset();
+    this.setState({
+      files: [],
+      errors: [],
+    });
+  }
+
+  validate(file) {
+    const ext = file.name.split('.').pop();
```

Comment:
be careful, you expect file always exists and is well formed. I think you should at least verify `file` is defined